src/wombats/handlers/helpers.clj

```diff
@@ -5,15 +5,22 @@
   "Throws an error that will be caught by the exception interceptor."
   [{code :code
     details :details
-    params :params,
+    params :params
+    message :message
+    field-error :field-error
     :or {code 1
          details {}
-         params []}}]
+         params []
+         message nil
+         field-error nil}}]
 
-  (let [message (get errors code "Oops, looks like something went wrong.")]
-    (throw (ex-info "Wombat Error" {:type :wombat-error
-                                    :message (->> params
-                                                  (into [message])
-                                                  (apply format))
-                                    :details details
-                                    :code code}))))
+  (let [message (or message
+                    (get errors code "Oops, looks like something went wrong."))]
```

Comment:
Not sure. This is the catch all error response, if wombat error happens to be thrown and no error is found, this will display. I believe this one is ok here, but if there are strong opinions I can move it.